bls/src/main/java/tech/pegasys/artemis/bls/mikuli/BLS12381.java

```diff
@@ -56,7 +55,7 @@
    * @return The Signature, not null
    */
   public static Signature sign(SecretKey secretKey, Bytes message) {
-    G2Point hashInGroup2 = new G2Point(hashToG2(message));
+    G2Point hashInGroup2 = new G2Point(HashToCurve.hashToG2(message));
```

Comment:
Would be good to restore the static import for `hashToG2`.

Reply:
Done.